Sources/JWTKit/ECDSA/ECDSAKey.swift

```diff
@@ -1,37 +1,8 @@
 @_implementationOnly import CJWTKitBoringSSL
 
 public final class ECDSAKey: OpenSSLKey {
-    public enum Curve: String, Codable {
```

Comment:
Can this be replaced with a typealias to ``JWK.Curve`` that is marked as deprecated/renamed to ensure that existing code doesn't break?